tests/integration/api/test_eda_credential.py

```diff
@@ -395,6 +395,41 @@ def test_delete_credential_used_by_activation_forced(
     assert default_activation.eda_credentials.count() == 0
 
 
+@pytest.mark.django_db
+def test_delete_credential_used_by_project_with_gpg_credential(
+    client: APIClient,
+    preseed_credential_types,
+):
+    gpg_credential_type = models.CredentialType.objects.get(
+        name=enums.DefaultCredentialType.GPG
+    )
+    eda_credential = models.EdaCredential.objects.create(
+        name="test_gpg_credential",
+        inputs={"gpg_public_key": "secret"},
+        credential_type=gpg_credential_type,
+        managed=True,
```

Comment:
@hsong-rh This should not have the managed=True this is a user defined EdaCredential.